app/v1/connectors/CreateAmendUkSavingsAnnualSummaryConnector.scala

```diff
@@ -39,12 +39,17 @@ class CreateAmendUkSavingsAnnualSummaryConnector @Inject() (val http: HttpClient
 
     implicit val successCode: SuccessCode = SuccessCode(Status.OK)
 
+    val path = s"income-tax/nino/${nino.nino}/income-source/savings/annual/${taxYear.asDownstream}"
+
     val downstreamUri =
       if (taxYear.useTaxYearSpecificApi) {
         TaxYearSpecificIfsUri[Unit](s"income-tax/${taxYear.asTysDownstream}/$nino/income-source/savings/annual")
+      } else if (featureSwitches.isDesIf_MigrationEnabled) {
+        IfsUri[Unit](path)
       } else {
-        DesUri[Unit](s"income-tax/nino/${nino.nino}/income-source/savings/annual/${taxYear.asDownstream}")
+        DesUri[Unit](path)
       }
```

Comment:
Rather than declaring the `path` outside the code block (and have it not used at all for the TYS case) maybe declare it closer:
```
      } else {
        val path = s"income-tax/nino/${nino.nino}/income-source/savings/annual/${taxYear.asDownstream}"

        if (featureSwitches.isDesIf_MigrationEnabled) IfsUri[Unit](path) else DesUri[Unit](path)
      }
```